pkg/plugins/leader/postgres/leader_elector_test.go

```diff
@@ -11,39 +8,22 @@ import (
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 
-	"github.com/kumahq/kuma/pkg/config"
-	"github.com/kumahq/kuma/pkg/config/plugins/resources/postgres"
 	"github.com/kumahq/kuma/pkg/core/runtime/component"
 	common_postgres "github.com/kumahq/kuma/pkg/plugins/common/postgres"
 	leader_postgres "github.com/kumahq/kuma/pkg/plugins/leader/postgres"
 	"github.com/kumahq/kuma/pkg/test"
+	pg_test "github.com/kumahq/kuma/pkg/test/store/postgres"
```

Comment:
nit: looks like the import alias convention would be `test_postgres`?

Reply:
Is it something quite systematic to have these types of aliases?